stable/kube-state-metrics/templates/monitor.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
       release: "{{ .Release.Name }}"
       heritage: "{{ .Release.Service }}"
       {{- if .Values.prometheus.monitor.additionalLabels }}
-{{ toYaml .Values.prometheus.monitor.additionalLabels | indent 4 }}
+{{ toYaml .Values.prometheus.monitor.additionalLabels | indent 6 }}
```

Comment:
I don't see any reason for there to be a `labels` value under the `labels` key field. We should fix that IMO and  we can then continue using `indent 4`.

Reply:
Please see line 8.

Reply:
Oh I see, yea let me fix that.

Reply:
Fixed.